index.js

```diff
@@ -23,18 +26,34 @@ exports.register = (server, options, next) => {
 
     // update all hooks:
     const updateHooks = require('./lib/updateHooks.js');
-
     const hook = require('./lib/hook.js');
+    const retry = require('./lib/retry.js');
 
     // register the 'hook' method with the server:
     if (options.decorate) {
       server.decorate('server', 'hook', (hookName, hookData, hookOptions) => {
         hook(server, settings, collection, hookName, hookData, hookOptions || {});
       });
+      server.decorate('server', 'retry', (hookId, callback) => {
```

Comment:
I think this should be retryHook or hookRetry.  Seems a little scary to add a `server.retry()`